settings.gradle

```diff
@@ -7,4 +7,8 @@ include ':org.eclipse.buildship.branding'
 include ':org.eclipse.buildship.core.test'
 include ':org.eclipse.buildship.ui.test'
 include ':org.eclipse.buildship.stsmigration'
-include ':org.eclipse.buildship.stsmigration.test'
\ No newline at end of file
+include ':org.eclipse.buildship.stsmigration.test'
+include ':org.eclipse.buildship.oomph'
+include ':org.eclipse.buildship.oomph.edit'
+include ':org.eclipse.buildship.oomph.feature'
+include ':org.eclipse.buildship.oomph.test'
```

Comment:
As mentioned above, how about that we include the oomph plugins when a system property is present?